Umweltzone/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,327 @@
+    <string name="city_info_section_title_lez">Detalhes sobre as zonas de baixa emissão</string>
+    <string name="city_info_section_title_dpz">Detalhes sobre as zonas de proibição de diesel</string>
+    <string name="city_info_section_title_other">Mais informações</string>
+    <string name="city_info_zone_status">disponível</string>
+
+    <string name="city_info_zone_number_as_of">
+        A partir de <xliff:g example="1 de Out de 2099" id="date">%1$s</xliff:g> apenas veículos com um adesivo
+        <xliff:g example="na cor verde" id="with_colors">%2$s</xliff:g> têm permissão
+        para transitar em zonas de baixa emissão.
+    </string>
+    <string name="city_info_zone_number_since">
+        Desde <xliff:g example="1 de Fev de 2016" id="date">%1$s</xliff:g> apenas
+        veículos com adesivos <xliff:g example="na cor verde" id="with_colors">%2$s</xliff:g> têm permissão
+        para transitar em zonas de baixa emissão.
+    </string>
+
+    <string name="city_info_zone_number_since_date_format">d de MMM de yyyy</string>
+    <string name="city_info_zone_number_since_text_fragment_green">na cor verde</string>
+    <string name="city_info_zone_number_since_text_fragment_yellow">na cor amarelo ou verde</string>
+    <string name="city_info_zone_number_since_text_fragment_red">na cor vermelho, amarelo ou verde</string>
+    <string name="city_info_zone_number_none">Veículos de todas as classes de emissão têm permissão para transitar na zona de baixa emissão.</string>
+
+    <string name="city_info_next_zone_number_as_of">
+        A partir de <xliff:g example="1 de Fev de 2016" id="date">%1$s</xliff:g> apenas
+        veículos com adesivos <xliff:g example="na cor verde" id="with_colors">%2$s</xliff:g> têm permissão
+        para transitar em zonas de baixa emissão.
+    </string>
+    <string name="city_info_abroad_licensed_vehicle_zone_info">
+        Os veículos registrados no exterior têm permissão para transitar na zona de baixa emissão
+        até<xliff:g example="31 de Jan de 2016" id="date">%1$s</xliff:g> com um adesivo
+        <xliff:g example="na cor amarelo ou verde" id="with_colors">%2$s</xliff:g>.
+    </string>
+    <string name="city_info_list_of_cities">
+        A zona abrange as seguintes cidades ou
+        distritos: <xliff:g example="Bochum, Castrop-Rauxel, Essen" id="cities">%1$s</xliff:g> e
+        <xliff:g example="Recklinghausen" id="last_city">%2$s</xliff:g>.
+    </string>
+
+    <string name="city_info_further_information">Mais informações online</string>
+    <string name="city_info_badge_online_title">Formulário de inscrição para adquirir os adesivos</string>
+
+    <string name="city_info_geometry_source_text">Fonte das coordenadas geométricas:
+        <xliff:g example="Geoportal Berlin / Umweltzone Berlin" id="geometrySource">%1$s</xliff:g>
+    </string>
+
+    <string name="city_info_geometry_updated_at_date_format">
+        d de MM de yyyy
+    </string>
```

Comment:
This crashes the app because this must be a formatting string which defines the country-specific date formatting. Legal characters are `d` for day, `M` for month and `Y` for year. See also: [Java 8 documentation](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html).

Reply:
Done!